openstf/feature_engineering/apply_features.py

```diff
@@ -149,185 +55,50 @@ def apply_features(data, minute_list=None, h_ahead=24):
                             np.random.uniform(0.7,1.7, 200)))
 
     """
-    if minute_list is None:
-        minute_list = []
 
-    lag_functions = generate_lag_functions(data, minute_list, h_ahead)
+    # Get lag feature functions
+    lag_feature_functions = generate_lag_feature_functions(feature_set_list, horizon)
 
-    # TODO it seems not all feature use the same convention
-    # better to choose one and stick to it
-    timedriven_functions = {
+    # Get timedrivenfeature functions
+    timedriven_feature_functions = {
```

Comment:
@JanMaartenvanDoorn maybe nice to gave a `generate`/`create` function for this as well like the other two. It seems like a similar thing you do.

Reply:
Yes I agree with consistency, however in this case the feature functions are simplydeclared and not generated using specific locic. In this case I think making a special 'generate' function would overcomplicate things.